src/Logo.jsx

```diff
@@ -1,31 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Logo = ({ src, alt, ...attributes }) => (
-  <img src={src} alt={alt} {...attributes} />
-);
-
-Logo.propTypes = {
-  src: PropTypes.string.isRequired,
-  alt: PropTypes.string.isRequired,
-};
-
-const LinkedLogo = ({
-  href,
-  src,
-  alt,
-  ...attributes
```

Comment:
I could only find 2 attributes being added to the logo throughout the codebase. One was `className="logo"`, which was being done *everywhere* the logo was being used. I figured just being able to write `<Logo` instead of `<Logo className="logo"` made more sense, so I moved that down into the component itself.

The other attribute was less straightforward. In `MobileHeader` *only* the LinkedLogo had `itemType="http://schema.org/Organization"`. It wasn't clear to me why that would only be in the mobile header. For now I just removed it, but I'd like to revisit this and come up with a better solution.